client/state/selectors/get-customize-url.js

```diff
@@ -13,7 +12,7 @@ import { getThemeCustomizeUrl } from 'calypso/state/themes/selectors';
  * @returns {string}           Customizer or Block Editor URL
  */
 export default function getCustomizeUrl( state, themeId, siteId ) {
-	return isSiteUsingFullSiteEditing( state, siteId ) || isSiteUsingCoreSiteEditor( state, siteId )
+	return isSiteUsingFullSiteEditing( state, siteId )
```

Comment:
Instead of replacing this selector, we are just removing it here (not replacing).  The reasoning being that we are no longer using this function in the case of FSE?  As we conditionally use an anonymous callback that calls `getSiteEditorUrl` in the changes made above to `theme-options.js`?

Would it make sense to still have the `is_fse_active` logic in this selector for more consistency?  Or is this also allowing us to have proper customizer support for universal themes?

Reply:
That's a good question. This selector is only to generate "theme-options" right now, but that might not always be the case and it would be easy to miss handling FSE sites.

We could add an isFSEActive argument to the selector so that is required data when generating the customize url--I'll give that a try.

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/57626/commits/ada889d6599c8a341e912dc4f101271e75bfc4f0